CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## NEXT - TBD
+## [0.4.2] - 2021-11-08
```

Comment:
There were more changes that happened this month. I have a PR that I need to merge once VPN works for me. That should update the CHANGELOG.

Reply:
done